Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "exonum-testkit"
-version = "0.0.0"
+version = "0.1.0"
```

Comment:
Is there a particular reason to change the crate version? As far as I can see, it's not yet published, so I'd prefer to bump the crate version on dedicated tagged commits concurrently with publishing to `crates.io`. (At least, this seems to be the best practice with JS / Npm; maybe, Rust has other conventions?)